index.html

```diff
@@ -1018,6 +1019,27 @@ <h1>Download</h1>
 						<div class="col-md-6"><img src="images/sbc_images/nanopir1.jpg" alt="NanoPi R1 photo" width="8" height="5" loading="lazy"></div>
 					</div>
 				</div>
+				<!-- Radxa Zero -->
+				<div id="radxa" class="toggleDiv single-project">
+					<div class="row"><div class="col-md-12 project-title"><h3>Radxa zero</h3><span class="show_hide close"></span></div></div>
+					<div class="row">
+						<div class="col-md-6 project-description">
+							<div class="project-info">
+								<div><span>CPU:</span>Amlogic S905Y2 1.8 GHz | 4 cores | ARMv8</div>
+								<div><span>RAM:</span>512 MiB - 4 GiB DDR4</div>
+								<div><span>USB:</span>USB 2.0 OTG and power, USB 3.0 HOST</div>
+								<div><span>NET:</span>Onboard WiFi 4/5 (802.11 a/b/g/n/ac), Bluetooth 4/5</div>
+								<div><span>Storage:</span>8 - 128 GiB eMMC</div>
+								<br>
+								<div><a href="https://dietpi.com/downloads/images/DietPi_RadxaZero-ARMv8-Bullseye.7z"><span>Download Image:</span><i class="fa fa-download"></i><u>Download</u></a></div>
+								<div><a href="https://dietpi.com/docs/install/" target="_blank" rel="noopener"><span>How to install / Details:</span><i class="fa fa-book"></i><u>View</u></a></div>
+							</div>
+							<p>Radxa Zero features a quad core 64 ARM processor, 32bit LPDDR4, up to 4K@60 HDMI, 802.11 ac WIFI, Bluetooth 5.0, USB 3.0, 40-pin GPIO header. Radxa Zero supports USB 2.0 OTG&power via one USB C port.<br>
+								Radxa Zero comes in 4 options with different ram/storage configuration.</p>
```

Comment:
Some possible capitalisation issues below: "WiFi" and "RAM" (capital letters) is needed to pass the spell check.

Reply:
It should be "WIFI", "RAM"?

Reply:
"WiFi" and "RAM":
- https://en.wikipedia.org/wiki/Wi-Fi (we omit the dash, so basically like in the logo)
- https://en.wikipedia.org/wiki/Random-access_memory